src/libraries/System.Security.Cryptography.X509Certificates/tests/CertificateCreation/PrivateKeyAssociationTests.cs

```diff
@@ -426,6 +426,7 @@ public static void AssociatePersistedKey_CNG_DSA()
         }
 
         [Fact]
+        [SkipOnPlatform(TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst, "DSA is not available of iOS/tvOS/MacCatalyst")]
```

Comment:
This triplet of OSes exists a lot.  Perhaps we should make a MobileAppleCrypto value in PlatformSupport.

Reply:
I'll do it but it's a general concern shared elsewhere (in other assemblies; for `PlatformDetection`, etc.) and I have already discussed it elsewhere that it would be good to have an "iOS-like" shorthands.